src/main/java/com/github/netelli/ProductsDAO.java

```diff
@@ -1,15 +1,18 @@
 package com.github.netelli;
 
 import com.github.netelli.model.Product;
+import org.apache.log4j.Logger;
 
 import java.sql.*;
 import java.util.ArrayList;
+import java.util.Comparator;
 import java.util.List;
 
 public class ProductsDAO implements AutoCloseable {
```

Comment:
About class in general - as I mentioned earlier - add logging for every method and explain to user what is going on, how many Products where selected, how many of them were deleted, etc.